float32 hub_vel1
float32 hub_vel2
float32 step_pos1
float32 step_pos2
float32 step_vel1
float32 step_vel2

